samples/test/two_step_test.py

```diff
@@ -107,12 +130,13 @@ def verify(
             pipeline_func=two_step_pipeline,
             mode=kfp.dsl.PipelineExecutionMode.V1_LEGACY
         ),
+        # Verify default pipeline_root with minio
         TestCase(pipeline_func=two_step_pipeline,
-                 verify_func=verify,
+                 verify_func=verify_with_default_pipeline_root,
                  mode=kfp.dsl.PipelineExecutionMode.V2_COMPATIBLE,
                  arguments={
-                     kfp.dsl.ROOT_PARAMETER_NAME: 'minio://mlpipeline/v2/artifacts'},
-                 )
```

Comment:
Overriding pipeline root is also a test case we want to cover, can we keep it too?

Reply:
done